src/figuresClasses.ts

```diff
@@ -1,11 +1,70 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
per task only limited set of the values is allowed here - please consider using union types